blackbox/utils/mixins.py

```diff
@@ -0,0 +1,38 @@
+from blackbox.config import Blackbox as CONFIG
+from blackbox.utils.logger import log
+
+
+class SanitizeReportMixin:
+    """
+    Mixin class helper to mask all sensitive credentials
+
+    Could handle any string with sensitive output
+    """
+
+    def sanitize_output(self, sensitive_output: str) -> str:
+        """ Replace all self.config credentials with in any str *** """
+
+        # first of all let's combine list of secrets from config
+        sensitive_words = []
+        mixed_secrets = [CONFIG.databases.values(), CONFIG.storage.values(),
+                         CONFIG.notifiers.values(), ]
+        for mixed_secret in mixed_secrets:
+            self.extract_secrets(sensitive_words, mixed_secret)
```

Comment:
I don't understand why we're doing half the processing in this function and half in the other. Why can't we just do 

```py
sensitive_words = self.extract_secrets()
```
And then build `mixed_secrets` and do all the iteration inside that helper function?

Right now, this approach with making an empty list and passing it back and forth between this function and the other one and making 3 different calls just seems unnecessarily complex to me, and I think we can do better.